oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileNodeStoreBuilder.java

```diff
@@ -283,7 +324,7 @@ public List<IndexStore> buildList(IndexHelper indexHelper, IndexerSupport indexe
     private IndexStoreFiles createdSortedStoreFiles() throws IOException, CompositeException {
         // Check system property defined path
         String sortedFilePath = System.getProperty(OAK_INDEXER_SORTED_FILE_PATH);
-        if (StringUtils.isNotBlank(sortedFilePath)) {
+        if (sortedFilePath != null && !sortedFilePath.isEmpty()) {
```

Comment:
This is not equivalent. If the string contains only whitespace, `StringUtils::isNotBlank` returns false, while this test here returns true, as `#isEmpty` tests that the String contains no characters.

Reply:
I'll revert this... I couldn't compile this part of the code due to a problem with the dependency.

(I don't think the usage of "StringUtils.isNotBlank" is intentional btw.)

Reply:
Fixed.